definitions/contested/json/CaseEvent/CaseEvent.json

```diff
@@ -822,6 +822,34 @@
     "ShowSummary": "Y",
     "ShowEventNotes": "N"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_uploadCaseFilesAwaitingJudicialResponse",
+    "Name": "Upload Case Files",
+    "Description": "Upload Case Files",
+    "DisplayOrder": 14,
+    "PreConditionState(s)": "generalApplicationAwaitingJudiciaryResponse",
+    "PostConditionState": "generalApplicationAwaitingJudiciaryResponse",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/case-orchestration/upload-contested-case-documents",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_uploadCaseFilesConsentAssignedToJudge",
```

Comment:
dropped that note to slack but I think it would be possible to have the same result using just one state that has `generalApplicationAwaitingJudiciaryResponse, consentedOrderAssignJudge` as precondition states and `*` as post condition state